sign/eddilithium3/example_test.go

```diff
@@ -0,0 +1,45 @@
+package eddilithium3_test
+
+import (
+	"fmt"
+
+	"github.com/cloudflare/circl/sign/eddilithium3"
+)
+
+func Example() {
+	// Generates a keypair.
+	pk, sk, err := eddilithium3.GenerateKey(nil)
+	if err != nil {
+		panic(err)
+	}
+
+	// (Alternatively one can derive a keypair from a seed,
+	// see NewKeyFromSeed().)
+
+	// Packs public and private key
+	var packedSk [eddilithium3.PrivateKeySize]byte
```

Comment:
I think this example must show the minimum code for generating and verfiying a signature.
Additionally, it can be created other `Example_` functions for Pack/Unpack functions, if needed.

Reply:
You never just generate a keypair, sign and then directly verify: you pack a key after generating and you unpack a key before verifying with it. It doesn't make sense to force the reader to look at a different place to learn about that when they need it anyway.

Reply:
Most of people just copy-paste whatever they found in the examples, without paying attention to the details.

Reply:
Yes, I agree, and then it is nice that the packing/unpacking code is in there as well, isn't it?

Reply:
Yes, you can do that using another example function.

Reply:
But that would mean that the user would have to look in two different places.  I would prefer one example to be exhaustive for the common use case.  Packing and unpacking is required in the common use case.